src/data/finsupp/basic.lean

```diff
@@ -622,8 +628,8 @@ variables [add_monoid M]
 
 instance : has_add (α →₀ M) := ⟨zip_with (+) (add_zero 0)⟩
 
-@[simp] lemma add_apply {g₁ g₂ : α →₀ M} {a : α} : (g₁ + g₂) a = g₁ a + g₂ a :=
-rfl
+@[simp] lemma coe_add (f g : α →₀ M) : ⇑(f + g) = f + g := rfl
```

Comment:
Does this parse as ` ⇑(f + g) = ⇑f + ⇑g`?

Reply:
Yes.